app/controllers/facility_accounts_controller.rb

```diff
@@ -39,9 +16,9 @@ def configure_new_account(account)
   before_filter :init_current_facility
   before_filter :init_account
 
-  authorize_resource :class => Account
+  authorize_resource class: Account
 
-  before_filter :check_billing_access, :only => billing_access_checked_actions
+  before_filter :check_billing_access, only: check_billing_access_actions
```

Comment:
`check_billing_access_actions` is now populated by a class variable so multiple engines can concat it.